### PR TITLE
Governance tests hh fix

### PR DESCRIPTION
**Pull request by tjcloa**
Hardhat tests are updated and refactored. 
It is time.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,8 +45,9 @@
     "prettier": "prettier --write {,*/**/}*.{js,test.js,sol}",
     "lint-contracts": "solhint contracts/**/*.sol",
     "test": "brownie test",
-    "test-js": "truffle test",
-    "coverage": "truffle run coverage --file='tests-js/'"
+    "test-js": "echo Executing pure Hardhat test cases ... && npx hardhat test && echo Executing test cases using Ganache ... && npx hardhat --config hardhat-ganache-tests.config.js --network ganache test",
```
Comment (powerhousefrank) [inline package.json:48]: You might have to add this: https://www.npmjs.com/package/hardhat in order for the current CI to work. The tests are failing due to hardhat package not found I believe. Could you please check that?

**Review by powerhousefrank — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (tjcloa) [reply package.json:48]: done

Comment (powerhousefrank) [reply package.json:48]: Could you push it here?

**Comment by powerhousefrank** [comment]
This looks good now for me. @tjcloa please do have a look and you can merge.

File: package.json (hunk 1 above)
Comment (tjcloa) [reply package.json:52]: done

**Pull request merged**